web-api/migration-terraform/main/lambdas/migrations/0026-hearing-remove-gsi1pk.test.js

```diff
@@ -0,0 +1,91 @@
+const { migrateItems } = require('./0026-hearing-remove-gsi1pk.js');
+
+describe('migrateItems', () => {
+  let mockDocketEntry;
```

Comment:
World's largest nit: I think this variable is more like `mockHearing`?

Reply:
Thanks! Copy/Pasta!